test/parallel_api/ranges/std_ranges_test.h

```diff
@@ -660,14 +650,14 @@ struct test_range_algo
 #endif
             {
                 test<T, usm_vector<T>, mode, DataGen1, DataGen2>{}(
-                    n_device, dpcpp_policy<call_id + 10>(), algo, checker, subrange_view, subrange_view, args...);
+                    n_device, TestUtils::get_dpcpp_test_policy<class Test, call_id + 10>(), algo, checker, subrange_view, subrange_view, args...);
                 test<T, usm_subrange<T>, mode, DataGen1, DataGen2>{}(
-                    n_device, dpcpp_policy<call_id + 30>(), algo, checker, std::identity{}, std::identity{}, args...);
+                    n_device, TestUtils::get_dpcpp_test_policy<class Test, call_id + 30>(), algo, checker, std::identity{}, std::identity{}, args...);
 #if TEST_CPP20_SPAN_PRESENT
                 test<T, usm_vector<T>, mode, DataGen1, DataGen2>{}(
-                    n_device, dpcpp_policy<call_id + 20>(), algo, checker, span_view, subrange_view, args...);
+                    n_device, TestUtils::get_dpcpp_test_policy<class Test, call_id + 20>(), algo, checker, span_view, subrange_view, args...);
                 test<T, usm_span<T>, mode, DataGen1, DataGen2>{}(
-                    n_device, dpcpp_policy<call_id + 40>(), algo, checker, std::identity{}, std::identity{}, args...);
+                    n_device, TestUtils::get_dpcpp_test_policy<class Test, call_id + 40>(), algo, checker, std::identity{}, std::identity{}, args...);
```

Comment:
Here, if we switch the order, we can omit the repeated `class Test`.

Reply:
Fixed